src/sentry/middleware/integrations/parsers/base.py

```diff
@@ -14,6 +14,7 @@
 
 from sentry.api.base import ONE_DAY
```

Comment:
probably easier to review this file as it shows the diff of the additions i made. the `integrations/middleware/hybrid_cloud/parser.py` file is a copy of this.